css/amp.css

```diff
@@ -564,6 +564,7 @@ amp-iframe iframe {
 /**
  * Forms error/success messaging containers should be hidden at first.
  */
+form [submitting],
```

Comment:
validator file needs to be changed to support a similar  rule to `spec_name: "FORM > DIV [submit-success]"`